TESTS/API/I2C/I2C.cpp

```diff
@@ -85,13 +85,13 @@ void flash_WR()
 template<PinName sda, PinName scl, int address>
 void single_byte_WR()
 {
-    I2CEeprom memory(sda,scl,MBED_CONF_APP_I2C_EEPROM_ADDR,32,0);
+    I2CEEBlockDevice memory(sda,scl,MBED_CONF_APP_I2C_EEPROM_ADDR,32,0);
     char test = 'A' + rand()%26;
     char read;
     int r = 0;
     int w = 0;
-    w = memory.write(address,test);
-    r = memory.read(address,read);
+    w = memory.program((const void *)test, address, sizeof(test));

```

Comment:
`test` is not a pointer, why cast to void* ?

Reply:
> ARMmbed/i2cee-driver#6 is merged.
> But this I2CEeprom did not pass, can you fixed the test? @rajkan01

@jamesbeyond good catch. fixed the issue